app/views/includes/project-form.html

```diff
@@ -0,0 +1,52 @@
+<form class="form" id="projectForm">
+    <!-- Project name (title) -->
+    <div class="form-group">
+        <label class="form-label-bold" for="name">Title</label>
+        <p class="form-hint">Name of the project</p>
+        <input type="text" class="form-control" id="name" name="name" value="{{ project.name }}">
+    </div>
+
+    <!-- Location -->
+    <div class="form-group">
+        <label class="form-label-bold" for="location">
+            Location
+            <span class="form-hint">
+                Location of the team working on the project, e.g. London
+            </span>
+        </label>
+        <input type="text" class="form-control" id="location" name="location" value="{{ project.location }}">
+    </div>
+
+    <!-- Description -->
+    <div class="form-group">
+        <label class="form-label-bold" for="description">
+            Description
+            <span class="form-hint">
+                Description of the project
+            </span>
+        </label>
+        <textarea class="form-control" id="description" name="description">{{ project.description }}</textarea>
+    </div>
+
+    <!-- Phase -->
+    <div class="form-group">
+        <label class="form-label-bold" for="phase">
+            Phase
+            <span class="form-hint">
+                Phase of the project
+            </span>
+        </label>
+
+        <select class="form-control" name="phase" id="phase">
+            <option value="pipeline">Pipeline</option>
+            <option value="discovery">Discovery</option>
+            <option value="alpha">Alpha</option>
+            <option value="beta">Beta</option>
+            <option value="live">Live</option>
+        </select>
+    </div>
+
+    <div class="form-group">
+        <input type="submit" class="button" value="Continue">
```

Comment:
`Continue` suggests there is at least one more step before submitting your data to be persisted, maybe we could use something like `Update` or generic `Submit` instead?